### DISCUSSION TITLE
Unknown blade component

### DISCUSSION THREAD

**Discussion by patrickweh** [wireui/wireui#428] [Q&A]
Hey,

my phpstorm tells me that all wireui components are unknown, no autocomplete and warnings before pushing. Everything works but phpstorm doesnt recognize the components, does anyone know how to solve this?

Regards

**Comment by ph7jack** [comment]
@patrickweh I don't know any way to add an IDE autocomplete without using an extension. Maybe exists an extension to add blade components auto-complete. I don't know

**Reply by patrickweh** [reply]
when using the paid laravel idea plugin you can add an ide.json file to the top level of your project:

```{
    "$schema": "http://laravel-ide.com/schema/laravel-ide-v1.json",
    "blade": {
        "components": {
            "viewPaths": [
                {
                    "path": "vendor/wireui/wireui/resources/views/components"
                }
            ]
        }
    }
}
```

did the trick for me
